### ISSUE TITLE
Getting Started Guide

### ISSUE THREAD

**Issue by ethagnawl** [warhammerkid/bluetti_mqtt#53]
First off, thank you for putting this project together, @warhammerkid.

I've been experimenting with using this library -- possibly in order to integrate it into my Home Assistant setup. However, in trying to get a POC stood up, I ran into some blocking issues. I think it'd be useful for either the README or a second document to contain a tutorial which makes no assumptions and helps people get up and running. I'm happy to take a pass at this ... just as soon as I can get my POC working. 

My issue, assuming I'm even proceeding down the right track, has to do with the library interfacing with my MQTT broker. I'm able to detect and connect to my Bluetti devices without issue but the library isn't able to connect to the broker and reports, "Operation timed out" in an infinite loop. I've tried using a basic (no config aside from username and password) RabbitMQ broker in a Docker container and I see the following error in its logs when this library tries to connect:
 
```
mq_1  | 2022-12-13 15:09:12.354104+00:00 [info] <0.794.0> accepting AMQP connection <0.794.0> (172.29.0.1:33010 -> 172.29.0.2:5672)
mq_1  | 2022-12-13 15:09:12.354292+00:00 [error] <0.794.0> closing AMQP connection <0.794.0> (172.29.0.1:33010 -> 172.29.0.2:5672):
mq_1  | 2022-12-13 15:09:12.354292+00:00 [error] <0.794.0> {bad_header,<<16,54,0,4,77,81,84,84>>}

```

_Should_ this work or is there more configuration on the RabbitMQ side that I need to do? Alternatively, is there a preferred MQTT backend?

**Comment by warhammerkid** [comment]
I don't have experience using RabbitMQ and its [MQTT Plugin](https://www.rabbitmq.com/mqtt.html) as a broker, so I'm not sure exactly what's going on here. Are you sure you have the MQTT support set up correctly? I'd recommend trying to connect to it over MQTT with some other tool to verify that the issue is specifically with how bluetti_mqtt is connecting to it.

**Comment by ethagnawl** [comment]
> Are you sure you have the MQTT support set up correctly?

Nope. I was unaware that RabbitMQ required a separate plugin to facilitate MQTT and that's exactly the sort of thing I was thinking the "guide" would help people work around.

Anyways, your suggestion to try Mosquitto was very helpful and enough of a nudge to get my POC working. Thanks! If you're up incorporating something like what I'm outlining, I'll work on documenting my findings and provide some sample code that people can use to get started.
